libraries/CurieBLE/examples/LEDCentral/LEDCentral.ino

```diff
@@ -0,0 +1,157 @@
+/*
+  Copyright (c) 2016 Intel Corporation. All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-
+  1301 USA
+*/
+
+// This example can work with CallbackLED to show the profile read/write operation in central
```

Comment:
Are CallbackLED and LEDCentral required to work together? if so, this should be more obvious. Both sketches should have an explanation at the top, describing how the two sketches should be used together, and they should be named similarly, e.g. "LEDCentral" and "LEDPeripheral" or something like that.